chat_box/Readme.md

```
@@ -0,0 +1,27 @@
+tải thư viện uvicorn về, sau đó mở terminal ngay ở mục chat_box, terminal phải có đường mục giống thế này-"(.venv) C:\Users\LENOVO\myfilebro\diet-tracking\chat_box>",
+sau đó nhập lệnh uvicorn main:app --reload là server sẽ chạy
+trường hợp server không chạy, coi thử api của gemini đã load vào mô hình chưa, qua bên chat_bot_view.dart coi thử đường chạy local của mô hình đã đúng chưa
+##-----##
```

markdownlint-cli2 (0.18.1)

No space inside hashes on closed atx style heading

(MD020, no-missing-space-closed-atx)